agent/consul/rate/handler.go

```diff
@@ -134,6 +168,18 @@ func (h *Handler) UpdateConfig(cfg HandlerConfig) {
 	h.limiter.UpdateConfig(cfg.GlobalReadConfig, globalRead.ConfigKey())
 }
 
+func (h *Handler) GetGlobalReadLimiterConfig() (*multilimiter.LimiterConfig, bool) {
```

Comment:
Added this to be able to assert reload worked...is there a better way to assert this and not add this additional API surface area to handler and multilimiter?